examples/provider_with_oauth.tf

```diff
@@ -0,0 +1,12 @@
+provider "restapi" {
```

Comment:
Great to see an example added!

Can you please also include the `README.md` doco?

Reply:
@DRuggeri  Maybe we need some automatic doc generator ? as we are using this provider quite a lot at the moment i am happy to contribute. For now i will update the docs manual

Reply:
Yes, actually - if you are still interested, I started on the scaffolding to move the docs into the docs subdirectory to align with the expections for terraform models. If you'd like to give a whirl at just making the README a landing zone that directs users to those pages, that would be *awesome*.